src/border_agent/border_agent.cpp

```diff
@@ -165,6 +167,28 @@ BorderAgent::BorderAgent(otbr::Ncp::RcpHost &aHost, Mdns::Publisher &aPublisher)
     mHost.AddThreadStateChangedCallback([this](otChangedFlags aFlags) { HandleThreadStateChanged(aFlags); });
 }
 
+otbrError BorderAgent::CreateEphemeralKey(std::string &aEphemeralKey)
+{
+    std::random_device              r;
+    std::default_random_engine      engine(r());
+    std::uniform_int_distribution<> dist(0, 9); // ePSKc should be digits.
```

Comment:
Just a thought: maybe you can use `std::uniform_int_distribution<> dist('0', '9')` then you can get rid of the manual conversion into digits.

Reply:
accept, thanks!